### DISCUSSION TITLE
Add story description to individual story view

### DISCUSSION THREAD

**Discussion by brycenaddison** [storybookjs/storybook#28331] [Ideas]
### Describe the solution you'd like

The "Description" block for individual stories can be a nice feature when putting together a documentation page. However, I don't see why there wouldn't be an option to show the description configured for the story while viewing the story in the individual sandbox view. Perhaps something as simple as a question mark/help button in the toolbar showing the story description when clicked could do the trick.

### Describe alternatives you've considered

I've considered including story descriptions in a custom render method, so that it shows up on both the documentation page and the individual story sandbox. However, this solution feels clunky and unappealing.

### Are you able to assist to bring the feature to reality?

yes, I can

**Comment by anisjonischkeit** [comment]
Yes please! This would be super nice, I'd also be happy to assist in bringing this feature to reality.

I could imagine a thin strip above the panels section would be really nice. I had a look to see if there is a way to extend that panel with addons, but at a glance it looks like you can add panels, but you can't wrap the whole panels section.

It's sometimes hard to see exactly why you've got a certain story or why it shows in a certain way. You can often decipher it by reading what args have been set, but it's a few cognitive steps going from "hang on, why does this variation render like that, decipher the props, maybe read some code, ohhh, in this specific situation we want this behaviour"

**Comment by Vazaril** [comment]
Hey guys, i know its a bit late for you but i worked the last two days on a addon for storybook that adds a description section to the Toolbar and to the Addon panel. If you wanna check it out, here is the link to my Repo:
 https://github.com/Vazaril/storybook-addon-story-description
 And to the storybook integrations page:
 https://storybook.js.org/addons/storybook-addon-story-description
